docs/source/json-api/index.rst

```diff
@@ -200,9 +200,9 @@ start the standalone JAR, you can use the following command:
 
 .. code-block:: shell
 
-    java -jar http-json-1.5.0.jar --config json-api-app.conf
+    java -jar http-json-2.0.0.jar --config json-api-app.conf
 
-Replace the version number ``1.5.0`` by the version of the SDK you are
+Replace the version number ``2.0.0`` by the version of the SDK you are
```

Comment:
This is bound to become obsolete very fast. I would rather recommend to point at a generic `<version>` or something along those lines.